plugins/vpc-branch-eni/plugin/commands.go

```diff
@@ -280,9 +280,18 @@ func (plugin *Plugin) createVLANLink(branch *eni.Branch, linkName string, ipAddr
 			return err
 		}
 
+		// If gatewayIPAddress is nil, that means the gateway ip is not provided in NetConfig. In that case,
+		// use the gateway address inferred from the subnet.
+		if gatewayIPAddress == nil {
+			gatewayIPAddress = subnet.Gateways[0]
+			log.Infof("Using gateway ip inferred from subnet: %s", gatewayIPAddress.String())
```

Comment:
Minor: The gateway IP address is already logged at line 297. Whether that address came from netconfig is also logged after the netconfig is parsed. So I'm not sure whether this log statement is necessary, but I'm fine either way.